db/records.py

```diff
@@ -35,32 +36,22 @@ def get_records(
         order_by: list of SQLAlchemy ColumnElements to order by.  Should
                   usually be either a list of string column names, or a
                   list of columns from the given table.
-        filters:  list of tuples of type (ColumnElement, value), where
-                  ColumnElement is an SQLAlchemy ColumnElement, and value
-                  is a valid value for the associated column (i.e., the
-                  type must be correct)
+        filters:  list of dictionaries, where each dictionary has a 'field' and 'op'
```

Comment:
Can we document the list of `op` parameters somewhere? (or link to existing documentation)